v1/src/simulator/src/circuit.ts

```
@@ -66,76 +69,90 @@ export function resetScopeList() {
+    // globalScope.fixLayout();
+    scheduleBackup()
+    if (id === globalScope.id) return
+
+    // Now that we know we're switching to a different circuit, update verilog mode
+    if (!scopeList[id].verilogMetadata.isVerilogCircuit) {
+        verilogModeSet(false)
+        simulatorMobileStore.isVerilog.value = false
+    }
+
+    // $(`.circuits`).removeClass('current')
+    circuit_list.value.forEach((circuit) =>
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)